lib/_components/appFooter/template.njk

```diff
@@ -2,7 +2,7 @@
 	<div class="nhsuk-footer
 		{%- if params.classes %} {{ params.classes }}{% endif %}" id="nhsuk-footer"
 		{%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
-		<div class="nhsuk-width-container {%- if params.width %} {{ params.width }}{% endif %}">
+		<div class="nhsuk-width-container {%- if params.containerClasses %} {{ params.containerClasses }}{% endif %}">
```

Comment:
Don't need the if check.. can just be `{{ params.containerClasses }}`

Reply:
Do we not want to keep it consistent with how the other arguments are set up? They all have the if check, follows the NHSUK footer module too with the `{%- if params.classes %} {{ params.classes }}{% endif %}`